docs/api/http.md

```diff
@@ -293,37 +293,58 @@ The `Bun.serve` server can handle roughly 2.5x more requests per second than Nod
 ```ts
 interface Bun {
   serve(options: {
-    fetch: (req: Request, server: Server) => Response | Promise<Response>;
-    hostname?: string;
-    port?: number;
     development?: boolean;
-    error?: (error: Error) => Response | Promise<Response>;
+    error?: (request: ErrorLike) => Response | Promise<Response> | undefined | Promise<undefined>;
+    fetch(request: Request, server: Server): Response | Promise<Response>;
+    hostname?: string;
+    id?: string | null;
+    lowMemoryMode?: boolean;
+    maxRequestBodySize?: number;
+    port?: string | number;
+    reusePort?: boolean;
+    serverNames?: Record<string, TLSOptions>; // Server Name Indication (SNI) / TLS virtual hosts
```

Comment:
`serverNames` is incorrect, it does not exist in the codebase

`tls` can optionally be an array of the object type or just the object type

Reply:
Actually come to think of it I was checking types from `bun-types` version `1.1.4` not `main`. I'm checking `main` now.

Reply:
All set